recipes/hmat-oss/2bea755a2139907d331bfa0229154b6dab7a61c1.patch

```diff
@@ -0,0 +1,37 @@
+From 2bea755a2139907d331bfa0229154b6dab7a61c1 Mon Sep 17 00:00:00 2001
+From: Denis Barbier <bouzim@gmail.com>
+Date: Tue, 10 May 2016 12:47:02 +0200
+Subject: [PATCH] Instantiate abstract template classes *Function in
+ assembly.cpp
+
+This fixes issue #41.
+---
+ src/assembly.cpp | 15 +++++++++++++++
+ 1 file changed, 15 insertions(+)
+
+diff --git a/src/assembly.cpp b/src/assembly.cpp
```

Comment:
@jakirkham, yeah conda-build could be enhanced to try - p1 after -p0 patch strip level.